cmd/boots/dhcp.go

```diff
@@ -23,12 +23,13 @@ import (
 // ServeDHCP starts the DHCP server.
 // It takes the next server address (nextServer) for serving iPXE binaries via TFTP
 // and an IP:Port (httpServerFQDN) for serving iPXE binaries via HTTP.
-func ServeDHCP(addr string, nextServer net.IP, httpServerFQDN string) {
+func ServeDHCP(addr string, nextServer net.IP, ipxeBaseURL string, bootsBaseURL string) {
 	poolSize := env.Int("BOOTS_DHCP_WORKERS", runtime.GOMAXPROCS(0)/2)
 	handler := dhcpHandler{
-		pool:           workerpool.New(poolSize),
-		nextServer:     nextServer,
-		httpServerFQDN: httpServerFQDN,
+		pool:         workerpool.New(poolSize),
+		nextServer:   nextServer,
+		ipxeBaseURL:  ipxeBaseURL,
+		bootsBaseURL: bootsBaseURL,
```

Comment:
would naming this something that speaks to the ipxe script nature make sense? like `ipxeScriptURL` or `ipxeScriptBase` or similar?

Reply:
No objection here. I have a slight preference for ipxeBinariesURL, but not a huge one.

Reply:
nit: use singulars (as these values should only every be a singular location/name).
`bins` -> `bin`, `binaries`-> `binary`
`scripts` -> `script`

Reply:
They are singular `URL`s that hold multiple binaries and/or scripts...
Thus my original `xxxxBaseURL`

Reply:
Maybe the existing names are good enough...